settings/settings.go

```diff
@@ -89,6 +91,7 @@ func (s Settings) EphemeralDiskSettings() DiskSettings {
 	return DiskSettings{
 		VolumeID: s.Disks.Ephemeral,
 		Path:     s.Disks.Ephemeral,
+		ID:       s.Disks.Ephemeral,
```

Comment:
same thing here, if s.Disks.Ephemeral is a map[string]interface{} let's try to extract id, otherwise let's not set it just like for persistent disks.